src/main/java/reposense/util/FileUtil.java

```diff
@@ -41,6 +43,52 @@
 
     private static final String MESSAGE_INVALID_FILE_PATH = "\"%s\" is an invalid file path. Skipping this directory.";
 
+    /**
+     * Returns a list of paths to the generated reports from {@code repoConfigs}.
+     */
+    public static List<Path> getReportFolders(Path sourcePath, List<RepoConfiguration> repoConfigs) {
+        List<Path> reportFolders = new ArrayList<>();
+        for (RepoConfiguration repoConfig : repoConfigs) {
+            reportFolders.add(
+                    Paths.get(sourcePath + File.separator + repoConfig.getDisplayName()).toAbsolutePath());
+        }
+        return reportFolders;
+    }
+
+    /**
+     * Zips all files of type {@code fileTypes} that are in the directory {@code pathsToZip} into a single file and
+     * output it to {@code sourceAndOutputPath}.
+     */
+    public static void zipFoldersAndFiles(List<Path> pathsToZip, Path sourceAndOutputPath, String... fileTypes) {
+        zipFoldersAndFiles(pathsToZip, sourceAndOutputPath, sourceAndOutputPath, fileTypes);
+    }
+
+    /**
+     * Zips all files of type {@code fileTypes} that are in the directory {@code pathsToZip} into {@code outputPath}.
```

Comment:
`in the directory {@code pathsToZip}` seems to be an incorrect description.
The files are technically in `sourcePath`